services/minio/.env

```diff
@@ -8,3 +8,5 @@ MINIO_NUM_PARTITIONS=1
 # optionally define the host data folder for each partition (must be the same on each node)
 #MINIO_INTERNAL_VOLUME_DATA_PART0=/path/to/partition/0
```

Comment:
Justfiy commented. e.g. add "Example" in this case

Reply:
hmm you mean the line just above?

Reply:
MINOR: I meant that we need to distinguish between commented lines that are forgotten and those that are e.g. added as examples or notes. Please add some keyword as ``NOTE``, ``TODO`` or ``FIXME`` or ``EXAMPLE`` to avoid somebody from deleting it :-)